content/en/docs/reference/using-api/cel.md

```diff
@@ -0,0 +1,325 @@
+list type:
+
+- `set`: `X + Y` performs a union where the array positions of all elements in
+  `X` are preserved and non-intersecting elements in `Y` are appended, retaining
+  their partial order.
+- `map`: `X + Y` performs a merge where the array positions of all keys in `X`
+  are preserved but the values are overwritten by values in `Y` when the key
+  sets of `X` and `Y` intersect. Elements in `Y` with non-intersecting keys are
+  appended, retaining their partial order.
+
+## Escaping
+
+Only Kubernetes resource property names of the form
+`[a-zA-Z_.-/][a-zA-Z0-9_.-/]*` are accessible from CEL. Accessible property
+names are escaped according to the following rules when accessed in the
+expression:
+
+| escape sequence   | property name equivalent                                                                     |
+|-------------------|----------------------------------------------------------------------------------------------|
+| `__underscores__` | `__`                                                                                         |
+| `__dot__`         | `.`                                                                                          |
+| `__dash__`        | `-`                                                                                          |
+| `__slash__`       | `/`                                                                                          |
+| `__{keyword}__`   | [CEL **RESERVED** keyword](https://github.com/google/cel-spec/blob/v0.6.0/doc/langdef.md#syntax) |
+
+When you escape any of CEL's **RESERVED** keywords you need to match the exact property name
+use the underscore escaping
+(for example, `int` in the word `sprint` would not be escaped and nor would it need to be).
+
+Examples on escaping:
+
+| property name | rule with escaped property name   |
+|---------------|-----------------------------------|
+| `namespace`   | `self.__namespace__ > 0`          |
+| `x-prop`      | `self.x__dash__prop > 0`          |
+| `redact__d`   | `self.redact__underscores__d > 0` |
+| `string`      | `self.startsWith('kube')`         |
+
+## Resource Constraints
+
+CEL is non-Turing complete and offers a variety of production safety controls to
+limit execution time. CEL's _resource constraint_ features provide feedback to
+developers about expression complexity and help protect the API server from
+excessive resource consumption during evaluation. CEL's _resource constraint_
+features are used to prevent CEL evaluation from consuming excessive API server
+resources.
+
+A key element of the _resource constraint_ features is a "cost unit" that CEL
+defines as a way of tracking CPU utilization. "Cost units" are independant of
+system load and hardware. "Cost units" are also deterministic; for any given CEL
+expression and input data, evaluation of the expression by the CEL interpreter
+will always result in the same cost.
```

Comment:
Use emphasis only for the first use of a jargon-y term: _resource constraint_ and _cost unit_. Subsequent uses don't need emphasis.

Reply:
Thanks, I've also applied the treatment to _cost unit_.